### PR TITLE
chore: notify jira about contribution

### PR DESCRIPTION
**Pull request by OleksandrNechai**
[FX-2367]

### Description

Triggers JIRA automation on opened PR by external to FX contributor

### How to test

- Well, ask someone not in the FX team to create a PR

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2367]: https://toptal-core.atlassian.net/browse/FX-2367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/notify-jira-about-contribution.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  workflow_dispatch:
+  pull_request:
+    types: [opened]
+    branches:
+      - master
+
+jobs:
+  notify-jira:
+    # if external contribution
+    if: ${{!contains(fromJson('["denieler", "TomasSlama", "OleksandrNechai", "OndrejTuma", "dmaklygin1", "LashaJini"]'), github.event.pull_request.user.login)}}
```
Comment (LashaJini) [inline .github/workflows/notify-jira-about-contribution.yml:13]: Can we have something generic here? Cuz if someone new joins FX team, we have to update this each time.

**Review by TomasSlama — COMMENTED** [review]
super cool 👍 , will Jira know that it is generated from specific hook and will be able to add all settings needed?

File: .github/workflows/notify-jira-about-contribution.yml (hunk 1 above)
Comment (OleksandrNechai) [reply .github/workflows/notify-jira-about-contribution.yml:13]: Well, maybe, I tried to figure out how to read the contents of FX team on GitHub but did not find a simple way to do so. So I decided to not complicate things :)

Comment (dmaklygin1) [reply .github/workflows/notify-jira-about-contribution.yml:13]: maybe this GHA may help: https://github.com/tspascoal/get-user-teams-membership

Comment (OndrejTuma) [reply .github/workflows/notify-jira-about-contribution.yml:13]: I think we dont have generic information about contributors right now 🤔

**Comment by OleksandrNechai** [comment]
@TomasSlama yeah, this ticket is already generated by the automation https://toptal-core.atlassian.net/browse/FX-2367. Automation can be found here https://toptal-core.atlassian.net/jira/servicedesk/projects/FX/settings/automate#/rule/3021227

File: .github/workflows/notify-jira-about-contribution.yml (hunk 1 above)
Comment (OleksandrNechai) [reply .github/workflows/notify-jira-about-contribution.yml:13]: It seems to just check team membership

**Review by OndrejTuma — APPROVED** [review]

**Review by TomasSlama — APPROVED** [review]

**Pull request merged**